### ISSUE TITLE
training data is available?

### ISSUE THREAD

**Issue by lihuinian** [xypan1232/DislncRF#1]
Hi, Is it possible to get the training data of lncRNA-disease used in your study? thanks.

**Comment by xypan1232** [comment]
can you give me you mail, and i will send it to you. it should be noted that lncRNA-disease is not involved in model training, and it is an independent testing set.

**Comment by xypan1232** [comment]
I uploaded data_disease_doid.txt to this github, you can download it.

**Comment by PingPy** [comment]
Hi, Dr. Pan, I am reading your work and it is excellent. Is it possible send the datasets used in this article to me via Email? Great thanks. My email address is pengyao.ping2017@gmail.com

**Comment by PingPy** [comment]
Hi, Dr. Pan. I am reading your work which is excellent. could you send me the datasets used in this article via email? Great thanks! my email address is pengyao.ping2017@gmail.com.

**Comment by xypan1232** [comment]
disease-lncRNA pairs are data_disease_doid.txt in this github, but in this study, we only use disease-lncRNA as the test set, the trianing data is disease-associated PCGs, which is avaialble at https://github.com/xypan1232/DimiG/blob/master/data/human_disease_integrated_full.zip

**Comment by JappyPing** [comment]
> disease-lncRNA pairs are data_disease_doid.txt in this github, but in this study, we only use disease-lncRNA as the test set, the trianing data is disease-associated PCGs, which is avaialble at https://github.com/xypan1232/DimiG/blob/master/data/human_disease_integrated_full.zip

Thank you very much, Dr. Pan.

**Comment by meddoc0** [comment]
Hi , Dr.Pan
Thank you for reading!Could you send me the dataset and all the necessary file data about it？I couldn't find the files like string_9606___10_all_T.tsv , consolidated_all.tsv and so on.
My email is x2754728418@gmail.com , many Thanks!

**Comment by meddoc0** [comment]
Hi , Dr.Pan
Could you send me the dataset and all the necessary file data about this project ? I couldn't find the files like string_9606___10_all_T.tsv , consolidated_all.tsv and so on.
My email is x2754728418@gmail.com , many Thanks!
